src/main/java/uk/gov/mca/beacons/api/services/GetValidationErrorResponseService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.mca.beacons.api.services;
+
+import static uk.gov.mca.beacons.api.dto.ErrorResponseDTO.ValidationError;
+
+import org.springframework.stereotype.Service;
+import org.springframework.validation.Errors;
+import org.springframework.validation.FieldError;
+import uk.gov.mca.beacons.api.dto.ErrorResponseDTO;
+
+@Service
+public class GetValidationErrorResponseService
+  implements GetValidationErrorResponse {
```

Comment:
As above.  Believe this to be a mapper?

Reply:
Good spot 👍 

Have moved over into the `mappers` package and updated the interface/implementing class name to be inline with the other mappers